README.md

```diff
@@ -151,31 +162,32 @@ please feel free to use this basic working version.
 
 ### _rotary_ workflow summary
 
-1. Perform simple long read QC using
+1. Perform simple short read QC (e.g., score, length, and adapter trimming) using [bbduk.sh from bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
+2. Remove short read contamination using [bbduk.sh from bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
+3. Perform simple long read QC using
    [reformat.sh from bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/reformat-guide/)
-2. Assemble long reads using [Flye](https://github.com/fenderglass/Flye)
-3. Perform end repair of any circular contigs (using a heavily modified version of the
+4. Assemble long reads using [Flye](https://github.com/fenderglass/Flye)
+5. Perform end repair of any circular contigs (using a heavily modified version of the
    [circlator](https://github.com/sanger-pathogens/circlator) workflow)
-4. Polish contigs using long reads via [medaka](https://github.com/nanoporetech/medaka)
-5. If short read are provided, polishes using [Polypolish](https://github.com/rrwick/Polypolish)
+6. Polish contigs using long reads via [medaka](https://github.com/nanoporetech/medaka)
+7. If short reads are provided, polishes using [Polypolish](https://github.com/rrwick/Polypolish)
    and [POLCA](https://github.com/alekseyzimin/masurca)
-6. Filters resulting contigs by a user-provided coverage threshold
-7. Rotates any circular contigs to start at a marker gene of your choice (_dnaA_ by default), with help from
+8. Filters resulting contigs by a user-provided coverage threshold
+9. Rotates any circular contigs to start at a marker gene of your choice (_dnaA_ by default), with help from
    [circlator](https://github.com/sanger-pathogens/circlator)
-8. Performs one more round of polishing on circular contigs, either using
-   Polypolish (if short reads were provided) or medaka (if only long reads were provided)
-9. Gene prediction via [DFAST](https://github.com/nigyta/dfast_core)
-10. Functional and taxonomic annotation via [EggNOG-mapper](https://github.com/eggnogdb/eggnog-mapper)
+10. Performs one more round of polishing on circular contigs, either using
+    Polypolish (if short reads were provided) or medaka (if only long reads were provided)
+11. Gene prediction via [DFAST](https://github.com/nigyta/dfast_core)
+12. Functional and taxonomic annotation via [EggNOG-mapper](https://github.com/eggnogdb/eggnog-mapper)
     and [GTDB-Tk](https://github.com/Ecogenomics/GTDBTk)
 
 ### Known issues
 
-- Short read QC is not performed (you have to do it in advance)
 - Limited flags are exposed for some key tools in the pipeline (e.g., Flye)
 - Minimum supported length for circular contigs is 100 kb (it should be a fairly easy fix in a future version to
```

Comment:
This issue can be deleted now (addressed this when porting repair.py to python)

Reply:
Also, maybe can add another known issue? See below:

- If rotary is run in batch mode, then all samples in one batch must have only long reads, or all samples in one batch must have long and short reads. In future, we hope to make the pipeline more flexible in this area.

Reply:
Thanks for the additions.